### DISCUSSION TITLE
How to **transfer** a username?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#136001] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I know you can **change** a username. But what if I own 2 accounts, and want to swap usernames on them?
And be sure during this process no-one can steal it?

**Comment by GabrielAlmeidag** [comment, marked as answer]
When you change your GitHub username, the previous username is temporarily reserved and cannot be claimed by anyone else immediately. This protection allows you to safely swap usernames between two accounts without the risk of someone else taking your old username during the process. While GitHub doesn’t specify the exact duration of this reservation, it ensures that the previous username remains unavailable for a brief period, giving you time to complete the username changes on both accounts. To ensure everything goes smoothly, it’s best to complete the changes as quickly as possible.

**Reply by ghost** [reply]
Ok, thanks a lot @GabrielAlmeidag!
